src/main/java/com/github/swrirobotics/admin/AdminController.java

```diff
@@ -128,64 +128,64 @@ public String getAdminPage(Model model) {
     @RequestMapping(value = "/updateLatLons", method = RequestMethod.POST)
     @ResponseBody
     public void updateLatLons() {
-        myLogger.trace("updateLatLons");
+        myLogger.info("updateLatLons");
```

Comment:
I see all of the logger levels changed from trace to info. Was this intentional?

Reply:
Yes, mostly just because these are administrative functions that are only invoked through the admin panel in the UI.  It's useful to know every time they're called, and most people aren't going to go manually set the logging level to "trace".